models/jules/inst/template.job

```diff
@@ -20,7 +20,7 @@ if [ ! -e "@OUTDIR@/*.nc" ]; then
   # check the status
   if [ $STATUS -ne 0 ]; then
     echo -e "ERROR IN MODEL RUN\nLogfile is located at '@OUTDIR@/logfile.txt'" >&3
-    exit $STATUS
+#    exit $STATUS
```

Comment:
if there is an error the model should exit correct? So i think this should not be commented out.